xds/src/main/java/io/grpc/xds/CdsLoadBalancer2.java

```diff
@@ -225,8 +226,9 @@ private void handleClusterDiscovered() {
           childLb = null;
         }
         Status unavailable =
-            Status.UNAVAILABLE.withDescription("CDS error: found 0 leaf (logical DNS or EDS) "
-                + "clusters for root cluster " + root.name);
+            Status.UNAVAILABLE.withDescription(
```

Comment:
As long as you're updating it, it would be nice to switch to String.format() instead of concatenation

Reply:
Make sense. Updated to use `String.format()`.